css/styles.css

```diff
@@ -362,13 +394,58 @@ img {
   background-color: #141b22;
   width: 570px;
   padding: 82px 30px 30px 30px;
+  z-index: 1;
 }
 
+.highlighting::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  height: 50%;
+  border-width: 3px;
+  border-style: solid;
+  pointer-events: none;
+  border-image: linear-gradient(
+    to bottom right,
+    rgba(33, 231, 134, 1) 0%,
+    rgba(33, 231, 134, 0) 25%
+  )
+  1;
+opacity: 0;
+}
+
+.highlighting::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    width: 50%;
+    height: 50%;
+    border-width: 3px;
+    border-style: solid;
+    pointer-events: none;
+
+    border-image: linear-gradient(
+        to top left,
+        rgba(33, 231, 134, 1) 0%,
+        rgba(33, 231, 134, 0) 25%
+      )
+      1;
+    opacity: 0;
+  }
```

Comment:
Your styles for `.highlighting::after` and `.highlighting::before` almost the same (except `top, left` and `bottom, right `properties) - so you don't need to write the same styles, just make a group selector:
```
.highlighting::after,
.highlighting::before {...}
```